packages/plugins/rubygem-openscap/rubygem-openscap.spec

```diff
@@ -4,25 +4,19 @@
 
 %global gem_name openscap
 %global min_openscap_version 1.2.9
-%global max_openscap_version 1.3.8
+%global max_openscap_version 1.4
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.4.9
-Release: 8%{?dist}
+Release: 9%{?dist}
 Summary: A FFI wrapper around the OpenSCAP library
 Group: Development/Languages
 License: GPLv2+
 URL: https://github.com/OpenSCAP/ruby-openscap
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-# require libopenscap.so.8 in an arch neutral way
-Requires: openscap >= %{min_openscap_version}
-Requires: openscap < %{max_openscap_version}
-
-BuildRequires: openscap >= %{min_openscap_version}
-BuildRequires: openscap < %{max_openscap_version}
-BuildRequires: openscap-devel
-BuildRequires: bzip2
+# Loaded via FFI
+Requires: libopenscap.so.25
```

Comment:
This ends up pulling in both x86_64 and i686, but I'd like to avoid making it `libopenscap.so.25()(64bit)`. So not sure how you would best solve this. Perhaps just stick to our current way, but use `< 1.4` instead.

Reply:
There is no guarantee whatsoever that <1.4 will keep the soname.

Can we depend on the full path? Or use a macro like _isa to get the 64 bit stuff?

Reply:
I hope https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/VZINPQAWEZ25JUOIW34KH7FPKEHO36XB/ will provide something useful.

Reply:
https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/7TDDGFMI7G6HOWPLDVXBLIM3DB4QDVLD/ was very quick, but very useful. I've now pushed the change without a `%check` section, but that should at least let me verify if the require bit works as expected.

Reply:
Something isn't right because this still ends up without the suffix:
```console
# rpm -q --requires rubygem-openscap-0.4.9-9.el8.noarch.rpm
libopenscap.so.25
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
ruby
ruby(release)
ruby(rubygems)
ruby(rubygems)
rubygem(ffi) >= 1.0.9
rubygem(ffi) >= 1.0.9
```

Reply:
This RPM is noarch, which resets those macros

Reply:
You're right. Updated to copy what https://src.fedoraproject.org/rpms/rubygem-ruby-vips/blob/rawhide/f/rubygem-ruby-vips.spec#_29-30 does and verified on EL8 that it does what we want: pulls in only openscap.x86_64 and not openscap.i686.